nextflow.config

```diff
@@ -384,7 +384,7 @@ manifest {
     description     = 'Differential abundance analysis'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.3.0dev'
+    version = '1.3.0'
```

Comment:
Small nitpick: Algin spacing here

Reply:
Dammit- this is `nf-core bump-version`'s doing!